docs/api-docs/apps/tutorials/sample-app-next/05-step-3-integrate-api.md

```diff
@@ -0,0 +1,920 @@
+import { encodePayload, getBCAuth, setSession } from '../../lib/auth';
+```
+
+3. Update the logic to authenticate the app on install.
+
+```js
+export default async function auth(req: NextApiRequest, res: NextApiResponse) {
+    try {
+        // Authenticate the app on install
+        const session = await getBCAuth(req.query);
+        const storeHash = session?.context?.split('/')[1] || '';
+        const encodedContext = encodePayload(storeHash); // Signed JWT to validate/ prevent tampering
+
+        await setSession(session);
+        res.redirect(302, `/?context=${encodedContext}`);
+    } catch (error) {
+        const { message, response } = error;
+        res.status(response?.status || 500).json(message);
+    }
+}
+```
+
+[View code in GitHub](https://github.com/bigcommerce/sample-app-nodejs/blob/step-3-add-database/pages/api/auth.ts)
+
+**load**
+
+1. Open the `load.ts` file nested inside the `pages/api` folder.
+
+2. Import `encodePayload` and `setSession` from `/lib/auth`. Your imports should now look like this:
+
+```js
+import { NextApiRequest, NextApiResponse } from 'next';
+import { encodePayload, getBCVerify, setSession } from '../../lib/auth';
+```
+
+3. Update the logic to authenticate the app on load.
+
+```js
+export default async function load(req: NextApiRequest, res: NextApiResponse) {
+    try {
+        // Verify when app loaded (launch)
+        const session = await getBCVerify(req.query);
+        const storeHash = session?.context?.split('/')[1] || '';
+        const encodedContext = encodePayload(storeHash); // Signed JWT to validate/ prevent tampering
+
+        await setSession(session);
+        res.redirect(302, `/?context=${encodedContext}`);
+    } catch (error) {
+        const { message, response } = error;
+        res.status(response?.status || 500).json(message);
+    }
+}
+```
+
+[View code in GitHub](https://github.com/bigcommerce/sample-app-nodejs/blob/step-3-add-database/pages/api/load.ts)
+
+**uninstall** 
+
+1. Open the `uninstall.ts` file nested inside the `pages/api` folder.
+
+2. Import `getBCVerify` and `removeSession` from `/lib/auth`. Your imports should now look like this:
+
+```js
+import { NextApiRequest, NextApiResponse } from 'next';
+import { getBCVerify, removeSession } from '../../lib/auth';
+```
+
+3. Update the logic to delete the session on uninstall.
+
+```js
+export default async function uninstall(req: NextApiRequest, res: NextApiResponse) {
+    try {
+        const session = await getBCVerify(req.query);
+
+        await removeSession(res, session);
+        res.status(200).end();
+    } catch (error) {
+        const { message, response } = error;
+        res.status(response?.status || 500).json(message);
+    }
+}
+```
+
+[View code in GitHub](https://github.com/bigcommerce/sample-app-nodejs/blob/step-3-add-database/pages/api/uninstall.tss)
+
+## Add the Products endpoint
+
+You will use this endpoint to retrieve your products summary.
```

Comment:
We might want to specify that the products summary is coming from the [v3 catalog API](https://developer.bigcommerce.com/api-reference/store-management/catalog/summary/getcatalogsummary) so the devs can follow along and reference the API docs.  

Additionally, we might should mention somewhere that V3 endpoints must be used over V2 per the [App Requirements](https://developer.bigcommerce.com/api-docs/apps/guide/requirements#functionality)

Reply:
Ah, I see we added the V3 Catalog link to the end of `Update the homepage` section... we might actually want to list that link sooner, where we first consume it here